db/migrate/20180521172343_create_offers.rb

```diff
@@ -0,0 +1,14 @@
+class CreateOffers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :offers do |t|
+      t.references :product
```

Comment:
usually we first list the database columns like "description" or "title" and we add references at the end.
So:
create_table :offers do |t|
    t.string :title
    t.text :description
    t.integer :suggested_price
    t.string :status

    t.references :product
    t.references :customer

    t.timestamps
end

Reply:
Ok, modified accordingly